### ISSUE TITLE
Repo dont get any update - use https://github.com/wonday/react-native-orientation-locker

### ISSUE THREAD

**Issue by MateusAndrade** [yamill/react-native-orientation#343]
Good morning folks.

Since I started using this project, I ended up having a problem doing an android pro build because the compileSDK of this project is outdated. If it is necessary to edit it manually, or perform a fork of that project by changing that version.

Since no update has been made on this project for about a year, I think a valid option is to use this fork [react-native-orientation-locker](https://github.com/wonday/react-native-orientation-locker).
Basically it is the same project, but with this problem of Android solved and a higher rate of updates.

If you are having problems related to Android, IOS, or etc strongly recommend using this fork.

If possible, please forward this repo to [react-community](https://github.com/react-community) @yamill . Since this way, it will probably have more support, etc.

There are some issues/pr related to this compileSdk error: #334 , #335 .
